armbian/armbian-build.sh

```diff
@@ -44,8 +52,10 @@ case ${ACTION} in
 		cp -aR base/* armbian-build/userpatches/overlay/					# copy scripts and configuration items to overlay
 		cp -aR ../bin/go/* armbian-build/userpatches/overlay/bin/go			# copy additional software binaries to overlay
 
+		# TODO(Stadicus): pin specific tag
```

Comment:
What tag are you referring to here?

Reply:
We should checkout a specific release tag once things are more stable, e.g. `sunxi-5.3` (see https://github.com/armbian/build/releases). 

For that, we also need to follow "Using alternate armbian builder repos and branches"
https://docs.armbian.com/Developer-Guide_Build-Preparation/#using-alternate-armbian-builder-repos-and-branches

Not a priority, though.